crates/sui-source-validation-service/Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde = { version = "1.0.144", features = ["derive"] }
 serde_json = "1.0.88"
 url = "2.3.1"
 
-sui-move.workspace = true
+sui-move = { workspace = true, features = ["all"] }
```

Comment:
change needed to compile / run tests inside `crates/sui-source-validation-service`

Reply:
I'm not sure if this will work, but if this for tests, is it worth adding a new dependency in `dev-dependencies` that has `features = ["all"]` instead? I'm hoping cargo will do the right thing there.

Reply:
looks like this works! Will put up in a follow up to avoid rebasing/restacking this